delta-lake/delta-24x/src/main/scala/com/nvidia/spark/rapids/delta/delta24x/GpuDelta24xParquetFileFormat.scala

```diff
@@ -88,15 +88,22 @@ case class GpuDelta24xParquetFileFormat(
     val maxDelVecScatterBatchSize = RapidsConf
       .DELTA_LOW_SHUFFLE_MERGE_SCATTER_DEL_VECTOR_BATCH_SIZE
       .get(sparkSession.sessionState.conf)
+    val delVecScatterTimeMetric = metrics(GpuMetric.DELETION_VECTOR_SCATTER_TIME)
+    val delVecSizeMetric = metrics(GpuMetric.DELETION_VECTOR_SIZE)
+
 
     (file: PartitionedFile) => {
       val input = dataReader(file)
       val dv = delVecs.flatMap(_.value.get(new URI(file.filePath.toString())))
-        .map(dv => RoaringBitmapWrapper.deserializeFromBytes(dv.descriptor.inlineData).inner)
+        .map(dv => {
+          delVecSizeMetric += dv.descriptor.inlineData.length
+          RoaringBitmapWrapper.deserializeFromBytes(dv.descriptor.inlineData).inner
+        })
```

Comment:
Nit: Style guide is to use braces for multi-line map, which also reduces boilerplate here, e.g.:
```suggestion
        .map { dv =>
          delVecSizeMetric += dv.descriptor.inlineData.length
          RoaringBitmapWrapper.deserializeFromBytes(dv.descriptor.inlineData).inner
        }
```